agir/donations/components/donationPage/InformationsPage.js

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect, useState, useRef } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import Helmet from "react-helmet";
+import useSWR from "swr";
+import { useHistory } from "react-router-dom";
+
+import { Theme } from "./StyledComponents";
+import CONFIG from "./config";
+import * as api from "./api";
+
+import Skeleton from "@agir/front/genericComponents/Skeleton";
+import PageFadeIn from "@agir/front/genericComponents/PageFadeIn";
+import Spacer from "@agir/front/genericComponents/Spacer";
+import InformationsStep from "./InformationsStep";
+import Breadcrumb from "./Breadcrumb";
+import AmountInformations from "./AmountInformations";
+import {
+  StyledIllustration,
+  StyledBody,
+  StyledPage,
+  StyledLogo,
+  StyledMain,
+  Title,
+} from "./StyledComponents";
+import { displayPrice } from "@agir/lib/utils/display";
+import { scrollToError } from "@agir/front/app/utils";
+
+import { routeConfig } from "@agir/front/app/routes.config";
+
+const InformationsPage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [errors, setErrors] = useState({});
+
+  const scrollerRef = useRef(null);
+
+  const { data: session } = useSWR("/api/session/");
+  const { data: sessionDonation } = useSWR("/api/session/donation/");
+
+  const history = useHistory();
+  const params = useParams();
+  const { search } = useLocation();
+  const urlParams = new URLSearchParams(search);
+
+  const type = params?.type || "LFI";
+  const groupPk = type !== "2022" && urlParams.get("group");
+  const amountParam = urlParams.get("amount") || 0;
+
+  const { data: group } = useSWR(groupPk && `/api/groupes/${groupPk}/`, {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+    revalidateOnReconnect: false,
+  });
+
+  let amountStepUrl = routeConfig.donations.getLink(params);
+  // Keep params in url
+  if (!!groupPk) {
+    amountStepUrl += `?group=${groupPk}`;
+  }
+
+  const externalLinkRoute =
+    CONFIG[type]?.externalLinkRoute || CONFIG.default.externalLinkRoute;
+
+  const [formData, setFormData] = useState({
+    // amounts
+    to: type,
+    amount: amountParam || sessionDonation?.donations?.amount,
+    type: sessionDonation?.donations?.type,
```

Comment:
ça fait pas doublon avec `to` ?

Reply:
là ça nécessite des commentaires si to c'est 2022, on peut comprendre à la limite, mais type si c'est mensuel pas mensuel ça devrait être un boleen et s'appeler "mensuel"

Reply:
Non c'est que le nommage rentre un peu en conflit ici.
Le `type` en front correspond bien à à `LFI` / `2022` donc à au destinataire du don `to` côté back.
Et le `type` de don côté back correspond à `S` / `M` soit mensuel ou non.

Reply:
C'est peut-être qu'il faut changer le back alors, notamment parce que type c'est déjà utilisé aussi par ailleurs dans les payments en back.